docs/examples/ambassador-ingress.md

```diff
@@ -1,186 +1,181 @@
-```
-
-Install the Ambassador [edgectl tool](https://www.getambassador.io/docs/latest/topics/using/edgectl/edge-control/) 
+# Install Ambassador Gateway on k0s
+
+You can configure k0s with the [Ambassador API
+Gateway](https://www.getambassador.io/products/api-gateway/) and a [MetalLB
+service loadbalancer](https://metallb.universe.tf/). With the extensible
+bootstrapping functionality Helm offers, this is as simple as adding the
+correct extensions to the `k0s.yaml` file during cluster configuration.
+
+## Use Docker for non-native k0s platforms
+
+With Docker you can run k0s on platforms that the distribution does not
+natively support (refer to [Run k0s in Docker](../k0s-in-docker.md)). Skip this
+section if you are on a platform that k0s natively supports.
+
+As you need to create a custom configuration file to install Ambassador
+Gateway, you will first need prepare to map that file into the k0s container
+and to expose the ports Ambassador needs for outside access.
+
+1. Run k0s under Docker:
+
+    ```sh
+    docker run -d --name k0s --hostname k0s --privileged -v /var/lib/k0s -p 6443:6443 docker.io/k0sproject/k0s:latest
+    ```
+
+2. Export the default k0s configuration file:
+
+    ```sh
+    docker exec k0s k0s default-config > k0s.yaml
+    ```
+
+3. Export the cluster config, so you can access it using kubectl:
+
+    ```sh
+    docker exec k0s cat /var/lib/k0s/pki/admin.conf > k0s-cluster.conf
+    export KUBECONFIG=$KUBECONFIG:<absolute path to k0s-cluster.conf>
+
+## Configure k0s.yaml for Ambassador Gateway
+
+1. Open the ``k0s.yml`` file and append the following extensions at the end:
+
+    ```yaml
+    extensions:
+      helm:
+        repositories:
+          - name: datawire
+            url: https://www.getambassador.io
+          - name: bitnami
+            url: https://charts.bitnami.com/bitnami
+        charts:
+          - name: ambassador
+            chartname: datawire/ambassador
+            version: "6.5.13"
+            namespace: ambassador
+            values: |2
+              service:
+                externalIPs:
+                - 172.17.0.2
+          - name: metallb
+            chartname: bitnami/metallb
+            version: "1.0.1"
+            namespace: default
+            values: |2
+              configInline:
+                address-pools:
+                - name: generic-cluster-pool
+                  protocol: layer2
+                  addresses:
+                  - 172.17.0.2
+    ```
+
+    **Note**: It may be necessary to replace the 172.17.0.2 IP with your local IP address.
+
+    This action adds both Ambassador and Metallb (required for LoadBalancers) with the corresponding repositories and (minimal) configurations. Be aware that the provided example illustrates the use of your local network and that you will want to provide a range of IPs for MetalLB that are addressable on your LAN to access these services from anywhere on your network.
```

Comment:
MetalLB seems to spell their name with capital LB.

Reply:
Fixed.